packages/react/src/hooks/useTemplateModule.ts

```diff
@@ -80,14 +82,24 @@ export const useTemplateModule = <
         )
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
             root = template.specification.previewUrl[environment]!;
```

Comment:
Instead of adding the extra else if block we could probably just add a ternary here and do something like:

```typescript
root = template.specification.previewUrl[environment === Environment.Local ? Environment.Development : environment]!;
```